components/templates/Article.tsx

```diff
@@ -74,11 +75,15 @@ const Article: React.FunctionComponent<Props> = props => {
 
             {/* Title and author */}
             <h1>{props.title}</h1>
-            {props.author && <p className="text-muted">By {props.author}</p>}
+            { props.authorSlug &&
```

Comment:
This can be fixed by our `prettier` config, I suggest running `npm run prettier`. I usually run it alongside running `npm run eslint-fix` or use an IDE extension to run `prettier` on every save.